tests/test.ts

```diff
@@ -1,37 +1,31 @@
-import { assert } from 'chai';
-import { db } from '../src/server/db';
-import { club } from '../src/server/db/schema';
+import { describe, expect, test } from '@jest/globals';
+import { db } from '@src/server/db';
+import { type InsertClub } from '@src/server/db/models';
+import { club } from '@src/server/db/schema';
 import { eq } from 'drizzle-orm';
 
-//Test to create an organization
-describe('This is to create a club on supabase', () => {
-  it('should create a new club', (done) => {
-    // Creating a raw club object
-    const newClub: typeof club.$inferInsert = {
-      name: 'TEST ORG',
+describe('This should create a club on supabase', () => {
+  test('Should create a new club', async () => {
+    const newCLub: InsertClub = {
       description: 'Computer science club at UTD',
+      name: 'TEST ORG',
     };
 
-    db.insert(club)
-      .values(newClub)
-      .returning()
-      .then((res) => {
-        assert(res.length > 0);
-        assert.equal(res[0]?.name, 'TEST ORG');
-        done();
-      })
-      .catch(done);
-  }).timeout(5000);
+    const returned = await db.insert(club).values(newCLub).returning();
+    expect(returned.length > 0).toBeTruthy();
```

Comment:
I'm assuming `returned` should contain only the entry that was inserted into the db. If that's the case I think it would be best to modify the condition to `returned.length == 0`

Reply:
Fixed to just be `returned.length === 1`